content/en/docs/concepts/policy/limit-range.md

```diff
@@ -55,15 +55,33 @@ The name of a LimitRange object must be a valid
 
 A `LimitRange` does **not** check the consistency of the default values it applies. This means that a default value for the _limit_ that is set by `LimitRange` may be less than the _request_ value specified for the container in the spec that a client submits to the API server. If that happens, the final Pod will not be schedulable.
 
+### Create a namespace
+
+Create a namespace so that the resources you create in this exercise are
```

Comment:
Since this is a concept page, would prefer to have mention namespace in the manifest file.

Reply:
We can have other task / tutorial page where we can create a particular namespace and create LimitRange object.

Reply:
Done! PTAL